driver/src/main/scala/core/nodeset.scala

```diff
@@ -505,16 +507,14 @@ case class CrAuthenticating(db: String, user: String, password: String, nonce: O
 }
 
 case class ScramSha1Authenticating(
-    db: String, user: String, password: String,
```

Comment:
That should be fixed, I noticed just after raising the PR then amended the commit and forced pushed to overwrite it, but you were too quick :)
Anyway think Github might be confused (because of the force push) as it is still showing these formatting changes